docs/pages/ktable.vue

```diff
@@ -210,8 +232,141 @@
       </DocsShow>
       <!-- eslint-enable -->
 
-    </DocsPageSection>
+      <!--Table with Default Sort-->
+      <h3>Table with Default Sort</h3>
```

Comment:
For the case consistency, more in another comment

```suggestion
      <h3>Table with default sort</h3>
```